src/pe.rs

```diff
@@ -224,6 +228,13 @@ impl<'data, 'file> Iterator for PeSectionIterator<'data, 'file> {
     }
 }
 
+impl<'data, 'file> PeSection<'data, 'file> {
+    fn raw_data(&self) -> &'data [u8] {
+        &self.file.data[self.section.pointer_to_raw_data as usize..]
+            [..cmp::min(self.section.virtual_size, self.section.size_of_raw_data) as usize]
```

Comment:
I think these indices would benefit from being pulled out and defined in locals :-p

Reply:
Done, and fixed a related bug.